packages/plugin-core/src/components/views/NoteGraphViewFactory.ts

```diff
@@ -141,6 +153,35 @@ export class NoteGraphPanelFactory {
             }
             break;
           }
+          case GraphViewMessageEnum.onGraphThemeChange: {
+            this.defaultGraphTheme = msg.data.graphTheme;
+            break;
+          }
+          case GraphViewMessageEnum.onRequestDefaultGraphTheme: {
+            let graphTheme = MetadataService.instance().getDefaultGraphTheme();
+            if (!graphTheme) {
+              const ABUserGroup = GRAPH_THEME_TEST.getUserGroup(
```

Comment:
I think it may be a bit too disruptive to put the A/B Test here - I'm not sure we should change the theme for existing users.  How about we tweak the experiment a little bit? 

When a _new_ user is installing dendron, we have the A/B group decide which default style they should get.  We save that in `MetadataService`. 

For existing users, we can show them a toast message to the effect of "hey - we have different styles for the graph view now, check it out!".  We can maybe put this in a separate A/B experiment group (or maybe worth showing to everyone?).  For the toast, I have some code (wip) that can do this.  See https://github.com/dendronhq/dendron/pull/2832/files#diff-78b40b72529a46738cf6b47c9cc89fb508a2af3ee230f27c250eb8825c5f02eb

Reply:
Updated A/B test only for new users. 
Todo: Toast for existing users

Reply:
Updated A/B test only for new users. 
Todo: Toast for existing users

Reply:
Added